feature/default_sets.go

```diff
@@ -47,6 +47,11 @@ var defaultSetDesc = setDesc{
 		SetInit:    {}, // I
 		SetNodeAnn: {}, // N
 	},
+	lnwire.ZeroFeeSecondLevelOptional: {
```

Comment:
Perhaps we should also remove the "legacy" anchors now?

Reply:
Removed.